packages/back-end/src/integrations/ClickHouse.ts

```diff
@@ -102,4 +102,7 @@ export default class ClickHouse extends SqlIntegration {
       );
     return `table_schema IN ('${this.params.database}')`;
   }
+  isAutoGeneratingFactTablesSupported() {
+    return true;
```

Comment:
There are other Clickhouse where their schema's are not going to match up, right?  So we want this to only be true for the growthbook-clickhouse datasource types.